proj/hps_accel/src/blocks.h

```diff
@@ -23,7 +23,7 @@
 
 namespace hps_accel {
 // Encodes 16 signed 8-bit values in a 4x4 matrix
-struct Matrix4x4 {
+struct Vector16 {
```

Comment:
The Matrix4x4 name seemed to make more sense to me so I might have to pick your brain about what is happening here

Reply:
The way we ended up using these is as a Vector of 16 elements rather than as a 4x4 Matrix. This was my "I feel silly" moment yesterday. Fortunately, the vector approach results in much simpler code, at least intially.